client/login/index.node.js

```diff
@@ -13,5 +14,15 @@ export default router => {
 		);
 	}
 
+	if ( config.isEnabled( 'login/magic-login' ) ) {
+		// Only do the basics for layout on the server-side
+		router(
+			'/log-in/link/use/:lang?',
+			setUpLocale,
+			redirectLoggedIn,
+			makeLayout,
+		);
+	}
```

Comment:
Why don't we want to render this page server-side?

If I disable Javascript in my browser and load http://calypso.localhost:3000/log-in/link/use I have now:

<img width="633" alt="old" src="https://user-images.githubusercontent.com/594356/30060300-0ec69086-9243-11e7-9784-fe47b6b7ff61.png">

Whereas https://wpcalypso.wordpress.com/log-in/link/use gives me:

<img width="633" alt="new" src="https://user-images.githubusercontent.com/594356/30060311-1f332e20-9243-11e7-8233-fc78a3a775de.png">

I understand that the `Finish Login` button would not work in that case, but I think presenting content to the user as soon as possible makes our app feel more responsive.

Reply:
> Why don't we want to render this page server-side?

There was an adverse reaction to doing the `page.replace` when this was SSR'd...

Without the separate server-site route, calls to `page.redirect` were throwing an error because `document` was undefined in that env. I'm def opening to finding another solution to that if you have any ideas!